app/models/concerns/common_metadata.rb

```diff
@@ -15,6 +15,11 @@ module CommonMetadata
     property :source_metadata,
              predicate: ::RDF::URI.new('http://library.princeton.edu/terms/source_metadata'),
              multiple: false
+    property :state,
+             predicate: ::RDF::URI.new('http://fedora.info/definitions/1/0/access/objState'),
+             multiple: false do |index|
```

Comment:
The Fedora 4 ontology is in RDF.rb, but not the Fedora 3 ontologies.  I'll add stub vocabs for it and the PUL/OpaqueNamespace terms.